lib/metasploit/framework/login_scanner/snmp.rb

```diff
@@ -17,6 +17,35 @@ class SNMP
         PRIVATE_TYPES        = [ :password ]
         REALM_KEY            = nil
 
+        # @!attribute retries
```

Comment:
YARD has advised that `@!attribute` is not meant to be used on `attr_*`.  Use comments directly instead:

```ruby
# The number of retries
#
# @return [Fixnum]
attr_accessor :retries
```